generateExamples.js

```diff
@@ -20,11 +20,14 @@ function generateTemplateExample(template) {
     );
 
     // run the docusaurus script to bootstrap the template in the examples folder
+    const command = template.endsWith('-typescript')
+      ? `${template} --typescript`
```

Comment:
doesn't it lead to "classic-typescript --typescript" instead of "classic --typescript"? Maybe both work?

Reply:
Oops, you're right, `classic-typescript --typescript` would fail